test/distributed/org/apache/cassandra/distributed/test/FrozenUDTTest.java

```diff
@@ -135,20 +133,11 @@ public void testDivergentSchemas() throws Throwable
             cluster.schemaChange("create table " + KEYSPACE + ".x (id int, ck frozen<a>, i int, primary key (id, ck))");
 
             cluster.get(1).executeInternal("alter type " + KEYSPACE + ".a add bar text");
-            try
-            {
-                cluster.coordinator(1).execute("insert into " + KEYSPACE + ".x (id, ck, i) VALUES (?, " + json(1, 2) + ", ? )", ConsistencyLevel.ALL,
-                                               1, 2);
-                cluster.coordinator(1).execute("insert into " + KEYSPACE + ".x (id, ck, i) VALUES (?, " + json(1, 1) + ", ? )", ConsistencyLevel.ALL,
-                                               1, 1);
-                cluster.get(2).flush(KEYSPACE);
-                Assert.fail("Expected an exception to be thrown.");
-            }
-            catch (Exception e)
-            {
-                // correct path
-                System.out.println(e);
-            }
+            cluster.coordinator(1).execute("insert into " + KEYSPACE + ".x (id, ck, i) VALUES (?, " + json(1, 1) + ", ? )", ConsistencyLevel.ALL,
+                                           1, 1);
+            cluster.coordinator(1).execute("insert into " + KEYSPACE + ".x (id, ck, i) VALUES (?, " + json(1, 2) + ", ? )", ConsistencyLevel.ALL,
+                                           2, 2);
+            cluster.get(2).flush(KEYSPACE);
```

Comment:
For Review:

iirc, the changes made here did not work for `main-5.0` and since CASSANDRA-19764 seems to still be in-progress, I reverted the change made by STAR-9850 with the idea that when CASSANDRA-19764 lands in Apache it will also be reflected in `main-5.0` after a subsequent rebase.